### PR TITLE
Move filepath module

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,105 @@
+/*
+<<<<<<< HEAD
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+=======
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+>>>>>>> c4b79cec52b638e48caa6eb2b73401af64798ae9
```
Comment (BuddhiWathsala) [inline build.gradle:6]: Conflict

File: filepath-ballerina/build.gradle [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+def targetNativeJar = file("$project.rootDir/${moduleName}-native/build/libs/${moduleName}-native-${project.version}.jar")
+
+task ballerinaBuild {
+    inputs.dir file(project.projectDir)
+    doLast {
+        def originalConfig = ballerinaConfigFile.text
+        def newConfig = ballerinaConfigFile.text.replace("@project.version@", project.version)
+        newConfig = newConfig.replace("@toml.version@", tomlVersion)
+        ballerinaConfigFile.text = newConfig
+
```
Comment (BuddhiWathsala) [inline filepath-ballerina/build.gradle:72]: Shall we add this configuration here https://github.com/ballerina-platform/module-ballerina-jwt/blob/6f13d4f53f58b65332c2219544130622fad00e96/jwt-ballerina/build.gradle#L114

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (ThishaniLucas) [reply build.gradle]: fixed

File: filepath-ballerina/build.gradle (hunk 2 above)
Comment (ThishaniLucas) [reply filepath-ballerina/build.gradle:72]: done

File: filepath-ballerina/build.gradle [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerina - Filepath Ballerina Generator'
+
+configurations {
+    jbalTools
+    filepathBallerinaBirDir
+    filepathBallerinaJar
```
Comment (Bhashinee) [inline filepath-ballerina/build.gradle:25]:
```suggestion
```

**Review by BuddhiWathsala — APPROVED** [review]
LGTM @Bhashinee please check.

File: filepath-native/build.gradle [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ */
+
+plugins {
+    id 'java'
+}
+
+description = 'Ballerina - Filepath Java Utils'
+
+dependencies {
+    compile group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
```
Comment (Bhashinee) [inline filepath-native/build.gradle:25]: Do we need this dependency? If not shall we remove it?

Comment (ThishaniLucas) [reply filepath-native/build.gradle:25]: We need this. We have BString related use cases in the module.

Comment (ThishaniLucas) [reply filepath-native/build.gradle]: done

**Review by Bhashinee — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**